### ISSUE TITLE
www-client/firefox-9999: Could not find libclang to generate rust bindings for C/C++.

### ISSUE THREAD

**Issue by okias** [pkulev/riru#48]
emerge -v1  firefox
```
www-client/firefox-9999::riru  USE="dbus gmp-autoupdate hwaccel jemalloc pulseaudio startup-notification system-jpeg system-libevent system-sqlite wayland wifi -bindist -custom-cflags -custom-optimization -debug -gtk2 -hardened (-neon) -pgo (-selinux) (-system-cairo) -system-libvpx -test" 
```
...
```
0:12.65 checking for libpulse... yes
 0:12.66 checking MOZ_PULSEAUDIO_CFLAGS... -D_REENTRANT
 0:12.68 checking MOZ_PULSEAUDIO_LIBS... -lpulse
 0:12.68 checking for rustc... /usr/bin/rustc
 0:12.68 checking for cargo... /usr/bin/cargo
 0:13.10 checking rustc version... 1.35.0
 0:13.16 checking cargo version... 1.35.0
 0:13.79 checking for rust target triplet... x86_64-unknown-linux-gnu
 0:14.07 checking for rust host triplet... x86_64-unknown-linux-gnu
 0:14.07 checking for rustdoc... /usr/bin/rustdoc
 0:14.10 checking for cbindgen... /usr/bin/cbindgen
 0:14.10 checking for rustfmt... not found
 0:14.40 checking for clang for bindgen... /usr/lib/llvm/8/bin/clang++
 0:14.53 checking for libclang for bindgen... not found
 0:14.53 ERROR: Could not find libclang to generate rust bindings for C/C++. Please install the necessary packages, run `mach bootstrap`, or use --with-libclang-path to give the path containing it.
 0:14.67 *** Fix above errors and then restart with\
 0:14.67                "./mach build"
```

**Comment by okias** [comment]
issue is still valid

**Comment by pkulev** [comment]
Closed by #47
